frontend/src/routes/Upload.tsx

```diff
@@ -1138,7 +1145,7 @@ const finishUpload = async (
             if (CONFIG.upload.workflow) {
                 body.append("workflowDefinitionId", CONFIG.upload.workflow);
             }
-            await ocRequest("/ingest/ingest", { method: "post", body: body });
+            await ocRequest("/ingest/ingest", { method: "post", body: body }, id);
```

Comment:
I would probably pass `id` to the above "add ACL" `ocRequest` as well. Currently it's not necessary, but maybe in the future there will be a use case where the special roles are also necessary when attaching the ACL.